react/features/base/dialog/actionTypes.js

```diff
@@ -0,0 +1,22 @@
+import { Symbol } from '../react';
+
+/**
+ * The type of Redux action which begins a request to open a dialog.
+ *
+ * {
+ *     type: BEGIN_DIALOG_REQUEST,
+ *     component: React.Component,
+ *     props: React.PropTypes.object
+ * }
+ *
+ */
+export const BEGIN_DIALOG_REQUEST = Symbol('BEGIN_DIALOG_REQUEST');
```

Comment:
I think in this case we do want a begin...end sequence. Imagine a dialog where user input is requested, or where multiple buttons are presented, the ended action could contain data about that result.